src/main.cpp

```diff
@@ -264,4 +273,86 @@ void runSteeringDisplay() {
 	while (1) {
 		display.run();
 	}
+}
+
+void buttonRead(int buttonPress){
+	int ledPos;
```

Comment:
I'm trying to understand what this is for, are you turning on the LEDs specific to each button for testing?

Reply:
I was trying to test if we can trigger individual LEDs when specific buttons are pressed because the button numbering on the shift registers does not coincide with their respective LEDs. For example, if the wipers button is input 3 on the 597, the wipers LED is output 2 on the 595